docker/solr/schema.xml

```diff
@@ -1123,8 +1118,16 @@
     <field name="country" type="string" indexed="true" stored="true" required="false" multiValued="false" />
     <field name="text" type="text_general" indexed="true" stored="false" required="false" multiValued="false" />
 
+    <!-- all the languages (we currently support) of OCR, mapped to the various
+         built-in language-specific field types -->
+    <field name="ocr_eng" type="text_en" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="ocr_fre" type="text_fr" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="ocr_spa" type="text_es" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="ocr_ita" type="text_it" indexed="true" stored="true" required="false" multiValued="false" />
+    <field name="ocr_ger" type="text_de" indexed="true" stored="true" required="false" multiValued="false" />
+    <copyField source="ocr_*" dest="ocr" />
```

Comment:
Just a note for the future, we are going to be needing to add an ocr_cze field for Nebraska Czech papers.  If we are using a managed-schema it might be easier to build a script that would allow posting additional fields like this.  Otherwise we'll probably have to change this file itself, which isn't a huge deal, but it does start diverging from openoni's master